js/iosrtc.js

```diff
@@ -191,16 +191,15 @@ function registerGlobals(doNotRestoreCallbacksSupport) {
 		navigator.mediaDevices = {};
 	}
 
-	navigator.getUserMedia                  = getUserMedia;
-	navigator.webkitGetUserMedia            = getUserMedia;
-	navigator.mediaDevices.getUserMedia     = getUserMedia;
-	navigator.mediaDevices.enumerateDevices = enumerateDevices;
-
 	// Restore Callback support
 	if (!doNotRestoreCallbacksSupport) {
 		restoreCallbacksSupport();
 	}
 
+	navigator.getUserMedia                  = getUserMedia;
```

Comment:
@hthetiot callbacks support wasn't working for me when I was trying to use. Had to make this change to make it work. I'll try to check the versions you mentioned and create a PR if this change is not there in the repository.

Reply:
Looks like we still set [`navigator.getUserMedia`](https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/6ab6f8258d59b667382293ba08bd502e351c0730/js/iosrtc.js#L185) before calling [`restoreCallbacksSupport();`](https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/6ab6f8258d59b667382293ba08bd502e351c0730/js/iosrtc.js#L192)


Will create the PR.